src/Core/src/Platform/Android/SliderExtensions.cs

```diff
@@ -63,6 +62,11 @@ public static async Task UpdateThumbImageSourceAsync(this SeekBar seekBar, ISlid
 				if (seekBar.IsAlive() && thumbDrawable != null)
 					seekBar.SetThumb(thumbDrawable);
 			}
+			else if (slider.Handler is SliderHandler sliderHandler)
+			{
+				seekBar.SetThumb(sliderHandler.defaultThumbDrawable);
```

Comment:
Can we load the default thumb drawable from the android resources? I'm sure the default drawable here can probably get pulled from the android resources

Reply:
Yea, I was initially trying with this: `seekBar.SetThumb(context.GetDrawable(Resource.Drawable.abc_seekbar_thumb_material)), but I couldn't make it revert back to a default color, but I figured it out

Reply:
"but I figured it out" - smart guy alert! 😆

Reply:
 😂 😂 😂